arrows/ocv/refine_detections_write_to_disk.cxx

```diff
@@ -153,6 +153,24 @@ ::refine( vital::image_container_sptr image_data,
     return detections;
   }
 
+  // Get input filename if it's in the vital_metadata
+  std::string filename = "";
+  auto md = image_data->get_metadata();
+  if( md && md->has(VITAL_META_IMAGE_FILENAME) )
+  {
+    // Get the full path, and then extract just the filename proper
```

Comment:
IIRC, there is a function in kwiversys that will extract just the file name. (GetFilenameName( path ) ) This handles system specific path separators.